src/accelerate/big_modeling.py

```diff
@@ -39,9 +39,7 @@ def init_empty_weights(include_buffers: bool = False):
     Args:
         include_buffers (`bool`, *optional*, defaults to `False`):
             Whether or not to also put all buffers on the meta device while initializing.
-
     Example:
-
```

Comment:
Leaves those poor lines alone :-)

Reply:
🤣 yeah don't know what happened there

Reply:
Fixed in aafea4a1e18ac6562bc678f3e607e0d2c5f89b38 ;)